Vagrantfile

```diff
@@ -1,18 +1,32 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+# BOX_IMG = "generic/ubuntu1810" # Working  
+BOX_IMG = "ubuntu/bionic64" # Unable to ssh
+# BOX_IMG = "generic/ubuntu1804" # Working
+# BOX_IMG = "generic/ubuntu2004" # ssh problem
+
 # Install tmux, virtualenv, and mariadb-server to support development
 $preProvision= <<SCRIPT
 # Prevent attempt to access stdin, causing dpkg-reconfigure error output
 export DEBIAN_FRONTEND=noninteractive
-apt-get install -y tmux virtualenvwrapper
+
+# Installation kept failing due to python 2.
+# Changing default python to 3.
+apt -y update
+apt -y upgrade
+apt -y install python3-pip
+update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+python -m pip install --upgrade pip
+python -m pip install virtualenvwrapper
 
 # As per instructions at https://downloads.mariadb.org/mariadb/repositories
 apt-get install -y software-properties-common
 apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8 2>&1
 add-apt-repository -y 'deb [arch=amd64,arm64,i386,ppc64el] http://mirror.lstn.net/mariadb/repo/10.4/ubuntu xenial main'
 apt-get update
 apt-get install -y mariadb-server
+apt-get install -y tmux virtualenvwrapper
```

Comment:
This code probably works but it seems to be running more apt-gets calls than necessary. Can you reorder it so that fewer apt-get calls are made? Perhaps by moving your changes to beneath the mariadb installation and consolidating all the installation commands into one apt-get command.